testsuite/tests/virtual_file_system/tc_empty_vfs.out

```diff
@@ -0,0 +1,4 @@
+Dumping the root directory of an empty VFS:
```

Comment:
We can't have empty directories in a git repository. So maybe the test should create that directory before.